main/src/library/Option.flix

```diff
@@ -154,6 +158,16 @@ namespace Option {
             }
         }
 
+    ///
+    /// If argument `x` is a `Some`, return the result of applying the function to the value inside `x`.
+    /// Otherwise return `None`.
+    ///
+    pub def flatMap(f: a -> Option[b] & e, x: Option[a]) : Option[b] & e =
```

Comment:
What was wrong with the old flatMap? Which also uses the same documentations style as the rest of the library? :)